cfme/configure/tasks.py

```diff
@@ -128,7 +135,8 @@ class tabs(View):  # noqa
 
         @View.nested
         class mytasks(Tab):  # noqa
-            TAB_NAME = "My VM and Container Analysis Tasks"
+            TAB_NAME = VersionPick({'5.9': 'My Tasks',
```

Comment:
**Required**

This duplicates the `TAB_NAME` from L144 in `myothertasks` class below.

Reply:
Its the same for 5.9 , but different on Version.lowest !
What is your suggestion here?

Reply:
**REQUIRED**: A nitpick, but please keep the order oldest-newest, top to bottom.

Reply:
Hi Milan, what is nitpick?,
I can't understand what you want me to change, 
Since i change it as the other classes like mothertasks and alltasks below are written!
Can you please let me know exactly what should i change?
thank you

Reply:
@mfalesni is asking for:
TAB_NAME = VersionPick({
            Version.lowest(): 'My VM and Container Analysis Tasks',
            '5.9': 'My Tasks'
        })

Reply:
Thank you! I change it to oldest-newest, top to bottom